app/src/main/java/org/oppia/app/walkthrough/topiclist/WalkthroughTopicListFragment.kt

```diff
@@ -12,14 +12,19 @@ import javax.inject.Inject
 
 /** The second slide for [WalkthroughActivity]. */
 class WalkthroughTopicListFragment : InjectableFragment(), TopicSummaryClickListener {
-  @Inject lateinit var walkthroughTopicListFragmentPresenter: WalkthroughTopicListFragmentPresenter
+  @Inject
+  lateinit var walkthroughTopicListFragmentPresenter: WalkthroughTopicListFragmentPresenter
```

Comment:
Ditto here and elsewhere.

Reply:
Ktlint is a bit different, it says to be on the next line. Also, Ben discuss the same in yesterday's meeting